tests/jenkins/acm-cluster-hub.groovy

```diff
@@ -0,0 +1,304 @@
+                        def mm = email =~ /(https:\/\/[^ \n]+)/
+                        acm_console = mm[0][1]
+                        echo "ACM console = ${acm_console}"
+                        def mmm = email =~ /ssh (.*)/
+                        ssh_location = mmm[0][1]
+                        echo "SSH = ssh ${ssh_location}"
+                    } catch(Exception ex) {
+                        echo "Could not parse email:"
+                        echo email
+                        echo ex.toString()
+                        throw ex
+                    }
+                }
+            }
+        }
+        
+        stage ('Wait to complete provision') {
+        	steps {
+				echo "Wait for 5 minutes for deployment to complete"
+				sleep 300 // seconds
+			}
+		}
+
+        stage('Confirm before retiring') {
+            when {
+                expression {
+                    return params.confirm_before_delete
+                }
+            }
+            steps {
+                input "Continue ?"
+            }
+        }
+        stage('Retire service from CF') {
+            environment {
+                uri = "${cf_uri}"
+                credentials = credentials("${opentlc_creds}")
+                admin_credentials = credentials("${opentlc_admin_creds}")
+                DEBUG = 'true'
+            }
+            /* This step uses the delete_svc_guid.sh script to retire
+             the service from CloudForms */
+            steps {
+                git 'https://github.com/redhat-gpte-devopsautomation/cloudforms-oob'
+
+                sh "./opentlc/delete_svc_guid.sh '${guid}'"
+            }
+            post {
+                failure {
+                    withCredentials([usernameColonPassword(credentialsId: imap_creds, variable: 'credentials')]) {
+                        mail(
+                            subject: "${env.JOB_NAME} (${env.BUILD_NUMBER}) failed retiring for GUID=${guid}",
+                            body: "It appears that ${env.BUILD_URL} is failing, somebody should do something about that.\nMake sure GUID ${guid} is destroyed.",
+                            to: "${notification_email}",
+                            replyTo: "${notification_email}",
+                            from: credentials.split(':')[0]
+                        )
+                    }
+                    withCredentials([string(credentialsId: rocketchat_hook, variable: 'HOOK_URL')]) {
+                        sh(
+                            """
+                            curl -H 'Content-Type: application/json' \
+                            -X POST '${HOOK_URL}' \
+                            -d '{\"username\": \"jenkins\", \"icon_url\": \"https://dev-sfo01.opentlc.com/static/81c91982/images/headshot.png\", \"text\": \"@here :rage: ${env.JOB_NAME} (${env.BUILD_NUMBER}) failed retiring ${guid}.\"}'\
+                            """.trim()
+                        )
+                    }
```

Comment:
rocketchat stuff can be removed in all pipelines

Reply:
Removed from this file. Will remove from all file next week.